visual/EB_Bluetooth_temperature_and_battery_level/frmMain.cs

```diff
@@ -104,10 +107,17 @@ private void btn_stop_log_Click(object sender, EventArgs e)
 
         private void tmr_logging_data_Tick(object sender, EventArgs e)
         {
-            Port_deviceTest.WriteLine("AT + TEMPERATURE");
-            Console.WriteLine("AT + TEMPERATURE");
-            Port_deviceTest.WriteLine("AT + BATTERYLEVEL");
-            Console.WriteLine("AT + BATTERYLEVEL");
+            try
```

Comment:
add try catch to break blocking but it **not working**

Reply:
thankiu alot, i will try to break out the blocking when bluetooth over